Duplicati/WebserverCore/Endpoints/V1/Backups.cs

```diff
@@ -79,7 +80,7 @@ public static void Map(RouteGroupBuilder group)
                     Include = y.Include,
                     Expression = y.Expression,
                 }),
-                TargetURL = x.Backup.TargetURL,
+                TargetURL = EncryptedFieldHelper.Decrypt(x.Backup.TargetURL),
```

Comment:
This looks like it will be decrypted automatically when leaving the `Connection` class?